package.json

```diff
@@ -6,60 +6,59 @@
   "scripts": {
     "start": "node $NODE_APP_OPTIONS app.js",
     "lint": "node_modules/.bin/eslint .",
-    "format": "node_modules/.bin/prettier-eslint '**/*.js' --list-different",
-    "format:fix": "node_modules/.bin/prettier-eslint '**/*.js' --write",
+    "format": "node_modules/.bin/prettier-eslint $PWD'/**/*.js' --list-different",
+    "format:fix": "node_modules/.bin/prettier-eslint $PWD'/**/*.js' --write",
     "test:unit": "npm run test:unit:_run -- --grep=$MOCHA_GREP",
     "test:acceptance": "npm run test:acceptance:_run -- --grep=$MOCHA_GREP",
     "test:unit:_run": "mocha --recursive --reporter spec $@ test/unit/js",
-    "test:acceptance:_run": "mocha --recursive --reporter spec --timeout 15000 --exit $@ test/acceptance/js",
+    "test:acceptance:_run": "mocha --exit --recursive --reporter spec --timeout 15000 $@ test/acceptance/js",
     "nodemon": "nodemon --config nodemon.json"
   },
   "dependencies": {
-    "async": "^3.1.0",
-    "bcrypt": "^3.0.6",
+    "async": "^3.2.0",
+    "bcrypt": "^4.0.1",
     "body-parser": "^1.19.0",
-    "celebrate": "^10.0.1",
-    "coffeescript": "^1.12.7",
-    "connect-mongo": "^3.0.0",
-    "csurf": "^1.10.0",
+    "celebrate": "^12.0.0",
+    "connect-mongo": "^3.2.0",
+    "csurf": "^1.11.0",
     "express": "^4.17.1",
-    "express-rate-limit": "^5.0.0",
-    "express-session": "^1.12.1",
-    "lodash": "^4.17.11",
-    "logger-sharelatex": "^1.7.0",
-    "marked": "^0.6.2",
-    "metrics-sharelatex": "^2.2.0",
-    "mongodb": "^3.2.7",
-    "nodemailer": "^6.2.1",
+    "express-rate-limit": "^5.1.1",
+    "express-session": "^1.17.0",
+    "lodash": "^4.17.15",
+    "logger-sharelatex": "^1.9.0",
+    "marked": "^0.8.0",
+    "metrics-sharelatex": "^2.5.0",
+    "mongodb": "^3.5.4",
+    "nodemailer": "^6.4.4",
     "nodemailer-sendgrid-transport": "^0.2.0",
     "pug": "^2.0.4",
-    "request": "^2.88.0",
-    "request-promise-native": "^1.0.7",
+    "request": "^2.88.2",
+    "request-promise-native": "^1.0.8",
     "settings-sharelatex": "^1.1.0",
-    "yn": "^3.1.0"
+    "yn": "^4.0.0"
```

Comment:
Another major version bump without a changelog, but I can't see this package causing us problems if it's even in use at all.

Reply:
https://github.com/sindresorhus/yn/releases/tag/v4.0.0 (I'm compiling notes on upgrades in https://docs.google.com/document/d/1X1tm5rCFs7VWz5PdxBB-ZkjjRskC1xMMdDOCmWM8SFs/edit)